pkg/api/v1/types.go

```diff
@@ -862,6 +863,8 @@ type PodSpec struct {
 	DNSPolicy DNSPolicy `json:"dnsPolicy,omitempty" description:"DNS policy for containers within the pod; one of 'ClusterFirst' or 'Default'"`
 	// NodeSelector is a selector which must be true for the pod to fit on a node
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" description:"selector which must match a node's labels for the pod to be scheduled on that node"`
+	// AntiAffinitySelectors are selectors which must not match against existing pods labels for the pod to fit on a node
+	AntiAffinitySelectors []labels.LabelSelector `json:"antiAffinitySelectors,omitempty" description:"list of selectors which must not match against existing pods labels for the pod to fit on a node"`
```

Comment:
@davidopp 

Thanks for the suggestion. This changeset is for the second use case you mentioned, for Pods to be able to conflict with other Pods that are already scheduled to a Node. Should I call it PodConflictSelectors?